changelog/2022-03-08.md

```diff
@@ -0,0 +1,30 @@
+
+## :wrench: March 8, 2022
```

Comment:
why are these duplicated?

Reply:
ahh it shouldn't be. i updated the filename from 03.08 to 03.09 and it just created a new version of the file but didn't delete. will fix.